ArmPkg/Drivers/ArmGicDxe/GicV3/AArch64/ArmGicV3.masm

```diff
@@ -119,4 +121,24 @@ ArmGicV3SetBinaryPointer PROC
         ret
 ArmGicV3SetBinaryPointer ENDP
 
+//UINT32
+//EFIAPI
+//ArmGicV3GetControlRegister (
+//  VOID
+//  );
+ArmGicV3GetControlRegister PROC
+        mrs     x0, ICC_CTLR_EL1
+        ret
+ArmGicV3GetControlRegister ENDP
+
+//VOID
+//EFIAPI
+//ArmGicV3SetControlRegister (
+//  IN UINT32         Value

```

Comment:
This is incorrect, right? It should take in a `UINTN`: https://github.com/tianocore/edk2/blob/5e5ca20bf772bc80ca71b996def6f8ebad2860bf/ArmPkg/Drivers/ArmGicDxe/GicV3/Arm/ArmGicV3.S#L90-L92

https://developer.arm.com/documentation/ddi0595/2021-06/AArch64-Registers/ICC-CTLR-EL1--Interrupt-Controller-Control-Register--EL1-